O/OpenSpiel/build_tarballs.jl

```diff
@@ -2,32 +2,32 @@
 # `julia build_tarballs.jl --help` to see a usage message.
 using BinaryBuilder, Pkg
 
-julia_version = v"1.6.0"
+julia_version = v"1.6.3"
 
 name = "OpenSpiel"
-version = v"0.3.6"
+version = v"1.1.0"
 
 # Collection of sources required to complete build
 sources = [
-    ArchiveSource("https://github.com/deepmind/open_spiel/archive/v0.3.1.tar.gz", "3d2a7d1c2fb29e3d0df6f70c8692cb6b922a840bd58a86a3e7ef25e509cdbccb"),
+    ArchiveSource("https://github.com/deepmind/open_spiel/archive/v1.1.0.tar.gz", "6aecf251d13b03355f5dac8fe8bf95da16b71b5c961b366affd456dcd691a2eb"),
     ArchiveSource("https://github.com/findmyway/dds/archive/v0.1.1.tar.gz", "fd13ee77feb5b5c3dfcc3333a0523266beb2a3d27715703cf508313af25306e5"),
-    ArchiveSource("https://github.com/abseil/abseil-cpp/archive/20200923.1.tar.gz", "808350c4d7238315717749bab0067a1acd208023d41eaf0c7360f29cc8bc8f21"),
+    ArchiveSource("https://github.com/abseil/abseil-cpp/archive/20211102.0.tar.gz", "dcf71b9cba8dc0ca9940c4b316a0c796be8fab42b070bb6b7cab62b48f0e66c4"),
     ArchiveSource("https://github.com/findmyway/hanabi-learning-environment/archive/v0.1.0.tar.gz", "6126936fd13a95f8cadeacaa69dfb38a960eaf3bd588aacc8893a6e07e4791a3"),
     ArchiveSource("https://github.com/findmyway/project_acpc_server/archive/v0.1.0.tar.gz", "e29f969dd62ba354b7019cae3f7f1dbfbd9a744687ea4a8f7494c2bb1ee87382"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 
-mv open_spiel-0.3.1 open_spiel
-mv abseil-cpp-20200923.1 open_spiel/open_spiel/abseil-cpp
+mv open_spiel-1.1.0 open_spiel
+mv abseil-cpp-20211102.0 open_spiel/open_spiel/abseil-cpp
 mv dds-0.1.1 open_spiel/open_spiel/games/bridge/double_dummy_solver
 mv hanabi-learning-environment-0.1.0 open_spiel/open_spiel/games/hanabi/hanabi-learning-environment
 mv project_acpc_server-0.1.0 open_spiel/open_spiel/games/universal_poker/acpc
```

Comment:
This avoids hardocoding versions numbers every time
```suggestion
mv open_spiel-* open_spiel
mv abseil-cpp-* open_spiel/open_spiel/abseil-cpp
mv dds-* open_spiel/open_spiel/games/bridge/double_dummy_solver
mv hanabi-learning-environment-* open_spiel/open_spiel/games/hanabi/hanabi-learning-environment
mv project_acpc_server-* open_spiel/open_spiel/games/universal_poker/acpc
```
But why are you moving these directories?

Reply:
Ah, thanks, that's because those dirs are hardcoded and assumed to be in the correct subfolder